ecl/hqlcpp/hqlhtcpp.cpp

```diff
@@ -14088,6 +14101,9 @@ ABoundActivity * HqlCppTranslator::doBuildActivityDedup(BuildCtx & ctx, IHqlExpr
         buildCompareMember(instance->nestedctx, "Compare", order, DatasetReference(dataset));
         buildHashOfExprsClass(instance->nestedctx, "Hash", order, DatasetReference(dataset), true);
 
+        bool bHFDwholerecord = false;
```

Comment:
yes.  I don't think it is in the exisiting code.  It shouldn't need to match the enumeration name.